unfoldingWord/translationCore
# Feature - #3296 - Has Home ToolCard use same progress bar as Tool page

#### This pull request addresses:

Addresses #3296 

Also fixes how paths in tests were not done right for running jest on Windows. Made path separator agnostic by using path.join()

#### How to test this pull request:

Login, pick project, pick tool, then go back to "Home". The progress bar there should display the same as on the Tool page: If percentage is < 25% it is in black in the middle, else white and in the middle of the filled part of the bar.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3615)
